### DISCUSSION TITLE
Cannot add a protected PDF document

### DISCUSSION THREAD

**Discussion by Sans84** [spatie/laravel-medialibrary#3690] [Q&A]
When I try to add a password-protected PDF file to the library, I get an exception indicating that the file is protected:
`This file requires a password for access`

**Comment by freekmurze** [comment]
I'd accept a PR that fixes this.

**Comment by gurpreetkaits** [comment]
Hi @freekmurze, It's working I tried.

@Sans84 any way to reproduce?

**Comment by Sans84** [comment]
Hey, @gurpreetkaits 
> Hi @freekmurze, It's working I tried.

What works?

> @Sans84 any way to reproduce?

Any document that is password protected for viewing

> I'd accept a PR that fixes this.

@freekmurze, I have solved this problem for now by catching the error when generating the conversion for PDF. But I'm not sure if this is a good solution for PR

**Comment by gurpreetkaits** [comment]
Hey, @Sans84

So, What I tried is, I uploaded the password protected pdf. And there was no error. And later on, after uploading done I accessed the pdf so, it all worked. 

I'm not sure if I'm missing your point somewhere. This is what I understood.
